modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CustomerPortal.macro

```diff
@@ -232,7 +244,7 @@ definition {
 	}
 
 	macro loginOktaToUpdateCP {
-		var userEmailAddress = "garrettz@mbopartners.com.broken";
+		var userEmailAddress = "dbace26d5dadb9eed597@uat.com.broken";
 		var userPassword = "q4F63ZEH3njnnA34S761";
```

Comment:
No need to take actions now, but I think it would be better if we define these email/password values in a .properties file. @MathausC